README.md

```diff
@@ -3,7 +3,8 @@
 View all of the NBA's combine anthro data in one place as well as (work in progress) create your own database of that data.
 
 View an [html table](http://www.shauritahutchins.com/nba-draft-combine-anthro/) of the combine data.
-
+Example from the website:
```

Comment:
```suggestion

Example from the website:
```
Just adding a line here between the prior line.

Reply:
Sure looks great, you can approve it!